### DISCUSSION TITLE
How do you guys deal with slow rebuilds?

### DISCUSSION THREAD

**Discussion by thingsneverchange** [nuxt/nuxt#8316] [Questions (Nuxt 2)]
Hi all!

I am working on a fairly large project. It sizes about 400mb (70+ components / 40+ pages) without _statics_. I always try to use a pure CSS with as minimum modules as possible, and no typescript to avoid any extra layer that slows down the rebuild. It used to take 15 seconds with typescript & SCSS, and now it's 4-5sec. I did some other tunings, but the max I can get was 4-5 seconds. 

It's still _slow_. Just so you know, It's a big pain for an active development. Does anyone have a problem with this?

**Comment by sugoidesune** [comment]
When I was debugging functions in the generate hooks, it got annoying.
I used the `.nuxtignore` file to focus only on the pages i needed to work on.
I think this could be a good option for you to ignore 35 pages and only work on the few that need changing.

**Reply by DLLluoluo** [reply]
Can you give an example of .nuxtignore file configuration? Because many files are involved when writing some complex functions, it will be troublesome to configure them one by one. thank you very much

**Comment by gplusdotgr** [comment]
@sugoidesune what would you ignore though if you've got plenty of devs working on different parts of the app?... 
PS: Yes it's slow and unpleasant when you're doing styling :(
